### PR TITLE
Added Geocoding and reverse geocoding script to basic scripts

### PR DESCRIPTION
**Pull request by AdityaJ7**
This script can be used to obtain latitude and longitude from an address and vice-versa.

### CODE REVIEW

File: Basic-Scripts/Geocoding and Reverse Geocoding/geocoding_and_reverse_geocoding.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import requests
+import sys
+
+# If you want your own private_token please refer the README file for this script.
+private_token = "fd00189046e6f6"
```
Comment (SKAUL05) [inline Basic-Scripts/Geocoding and Reverse Geocoding/geocoding_and_reverse_geocoding.py]:
```suggestion
private_token = "<ENTER PRIVATE TOKEN>"
```
No need to expose tokens

File: Basic-Scripts/Geocoding and Reverse Geocoding/geocoding_and_reverse_geocoding.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            print("Cant find what you where looking for.")
+
+    # If the user choose to Quit the program
+    elif choice.upper() == 'Q':
+        sys.exit("Thanks for using the script")
+
+    # To make sure only valid input is provided
+    else:
+        print("Please make a valid choice")
+        print()
```
Comment (SKAUL05) [inline Basic-Scripts/Geocoding and Reverse Geocoding/geocoding_and_reverse_geocoding.py:72]: Enter Sample Input Output Here

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
@AdityaJ7 Kudos
One of the finest PR's reviewed so far.

**Comment by AdityaJ7** [comment]
> @AdityaJ7 Kudos
> One of the finest PR's reviewed so far.

Thanks, @SKAUL05  I have made the requested changes.

**Review by SKAUL05 — APPROVED** [review]
Good to go @ankitdobhal

**Pull request merged**